From: =?utf-8?q?C=C3=A9dric_Boutillier?= <boutil@debian.org>
Date: Wed, 12 Oct 2022 13:14:46 -0300
Subject: remove git usage in source gemspec

Last-Update: 2021-01-09
Forwarded: not needed

only when we use source from Github instead of released gem
---
 eventmachine.gemspec | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/eventmachine.gemspec b/eventmachine.gemspec
index 1c561fa..21771cb 100644
--- a/eventmachine.gemspec
+++ b/eventmachine.gemspec
@@ -10,8 +10,8 @@ Gem::Specification.new do |s|
   s.authors = ["Francis Cianfrocca", "Aman Gupta"]
   s.email   = ["garbagecat10@gmail.com", "aman@tmm1.net"]
 
-  s.files = `git ls-files README.md CHANGELOG.md GNU LICENSE rakelib ext java lib docs`.split
-  s.test_files = `git ls-files tests examples`.split
+  s.files = ["README.md", "CHANGELOG.md", "GNU LICENSE"] + Dir.glob("{rakelib,ext,java,lib,docs}/**/*")
+  s.test_files = Dir.glob("{tests,examples}/**/*")
 
   s.extensions = ["ext/extconf.rb", "ext/fastfilereader/extconf.rb"]
 
@@ -39,5 +39,5 @@ using TCP/IP, especially if custom protocols are required.
 EOT
 
   s.rdoc_options = ["--title", "EventMachine", "--main", "README.md", "-x", "lib/em/version", "-x", "lib/jeventmachine"]
-  s.extra_rdoc_files = `git ls-files README.md docs`.split
+  s.extra_rdoc_files = ["README.md"]+Dir.glob("docs/**/*")
 end
